pkg/virt-launcher/virtwrap/network/network.go

```diff
@@ -134,8 +134,7 @@ func SetupNetworkInterfacesPhase2(vmi *v1.VirtualMachineInstance, domain *api.Do
 	return nil
 }
 
-// a factory to get suitable network interface
-func getNetworkClass(network *v1.Network) (NetworkInterface, error) {
+func getNewNetworkInterface(network *v1.Network) (NetworkInterface, error) {
```

Comment:
IMO this factory has no place at all, it does nothing in practice and returning an interface is just wrong.
But if we do not want to go that far, then I would just recommend to drop the `get` prefix. `newNetworkInterface` sounds enough.

Reply:
I share your opinion about the future fate of this "factory" that produces one instance of a struct with no fields. This PR is supposed to be a minor step forward, calling entities for what they are. Future PRs should actually change these entities and how they relate to each other.